scripts/computeEtag.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 <md5cmd> <md5Index>";
+    echo "macOS: $0 md5 4";
+    echo "Linux: $0 md5sum 1";
+    exit 0;
+fi
+
+MD5_CMD=$1
+MD5_IDX=$2
+
+OUTPUTSTR=""
+
+for ver in generated/v*
+do
+    OUTPUTSTR+="**$(basename $ver)**\n"
+    OUTPUTSTR+="|Filename|etag|\n"
```

Comment:
I think this whole file would be simpler in node.js personally; we're not using shell scripts anywhere else.